packages/lpa-questionnaire-web-app/src/controllers/contact-us.js

```diff
@@ -0,0 +1,13 @@
+const { VIEW } = require('../lib/views');
+const { renderView } = require('../util/render');
+
+async function getContactUs(req, res) {
```

Comment:
Doesn't look like this function needs to be async?

Reply:
Yeah looks like a mistake, will revert